packages/epics/src/agreements/components/agreement-form-payment-schedule.tsx

```
@@ -0,0 +1,223 @@
+type Milestone = {
+  percentage: string;
+  dateRange: DateRange;
+};
+
+export type FormValues = {
+  option: Option;
+  futureDate?: Date;
+  milestones: Milestone[];
+};
+
+type AgreementFormPaymentScheduleProps = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)